pep-0457.txt

```diff
@@ -14,12 +14,17 @@ Created: 08-Oct-2013
 Overview
 ========
 
-This PEP proposes a syntax for positional-only parameters in Python.
+This PEP proposes a notation for positional-only parameters in Python.
 Positional-only parameters are parameters without an externally-usable
 name; when a function accepting positional-only parameters is called,
 positional arguments are mapped to these parameters based solely on
 their position.
 
+This PEP is an Informational PEP describing the notation for use when describing
```

Comment:
Maybe wrap at 72, like the rest of the document?

Reply:
Done!